.github/workflows/zip.yml

```diff
@@ -9,8 +13,29 @@ jobs:
   build:
     runs-on: macOS-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
+    - name: Build
+      run: |
+        cd ZipBuilder
+        swift build -v
+
+  package:
+    needs: build
+    runs-on: macOS-latest
+    # Only run the `package` job for the cron job, not PRs.
+    if: github.event_name == 'schedule'
```

Comment:
Nice find!

Reply:
Thanks! Hopefully it works... got this from the `if` syntax:
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idif

and the "`github` Context" https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#github-context

hoping that the name matches with the event trigger here:

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule